src/rpc/blockchain.cpp

```diff
@@ -2530,6 +2532,8 @@ static RPCHelpMan dumptxoutset()
                     {RPCResult::Type::STR_HEX, "base_hash", "the hash of the base of the snapshot"},
                     {RPCResult::Type::NUM, "base_height", "the height of the base of the snapshot"},
                     {RPCResult::Type::STR, "path", "the absolute path that the snapshot was written to"},
+                    {RPCResult::Type::STR_HEX, "assumeutxo", "the hash of the UTXO set contents"},
```

Comment:
`assumeutxo` seems like a weird name for this result, why not something like `txoutset_hash`?

Reply:
Yeah, good point - thought I had done it for consistency with something else but I guess not.